irods-vfs-impl/src/main/java/org/irods/nfsrods/vfs/IRODSVirtualFileSystem.java

```diff
@@ -853,7 +884,7 @@ public NfsIdMapping getIdMapper()
     @Override
     public Inode getRootInode() throws IOException
     {
-        return toFh(1);
+        return toFh(getInodeNumber(ZONE_COLLECTION));
```

Comment:
Can't use `ZONE_COLLECTION`.

NFSRODS can present any collection as the root.

Reply:
From our discussion, we determined that since `ZONE_COLLECTION` was initialized to `rodsSvrConfig.getZone()`, leaving ZONE_COLLECTION here is fine